### DISCUSSION TITLE
Welcome to sButtons Discussions!

### DISCUSSION THREAD

**Discussion by shahednasser** [sButtons/sbuttons#1191] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. You can suggest ideas you have here or anything you would like to talk about. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  Feel free to introduce yourself below and discuss anything you think would be helpful. If you also need help with anything, not just related to sButtons in particular, you can talk about it as well. Let's build this community together!

**Comment by Harsh-GitCollab** [comment]
Hello, I am new to open source and this is the first project I've decided to contribute to (and I'm super excited!). I have taken taken up the snake button issue. But I just need some help getting started . In the readme.md file there is a link to css file . But in there the code is without proper identation so when copy and paste from it in vs code it kindof gets messy. And I have to manually ident . Am I doing it wrong ? Is there some other method that I don't know ? Pls help

**Reply by shahednasser** [reply]
Hello, @Harsh-GitCollab thank you for deciding to contribute to our repository! 

The CSS file you are mentioning is a minified file and should not be modified manually. It is compiled from less files in the src directory. Any changes you want to make should be in that directory and once you are done you compile the changes to create the CSS file. You can read more about it in the [contributing guidelines](https://github.com/sButtons/sbuttons/blob/d15202d2173c2e1c6105de7aa8eb8bf89d2b8c6f/CONTRIBUTING.md).

If you have any other questions please let me know!

**Comment by harshad71** [comment]
hey hi i m harshad new to open source but i got all the things 
so i m here to learn and contribute i want to become part of sButtons comunity and want to contribute 
so will please guide me 

thanks
